src/Symfony/Component/Messenger/Worker.php

```diff
@@ -282,4 +331,68 @@ public function getMetadata(): WorkerMetadata
     {
         return $this->metadata;
     }
+
+    public function handleFutures(string $transportName, $parallelProcessLimit): void
+    {
+        $toHandle = self::$futures;
+        self::$futures = [];
+
+        if (!$toHandle) {
+            return;
+        }
+
+        $futuresReceived = array_map(fn ($combo) => $combo[0], $toHandle);
+        $envelopesAssociated = array_map(fn ($combo) => $combo[1], $toHandle);
+
+        [$errorsFromAwait, $executions] = awaitAll($futuresReceived);
+
+        // TO EDIT : because I need to check the execution
```

Comment:
I need to edit this part because I need to verify the execution and check if it's actually an error from the Handler